openmetadata-ui/src/main/resources/ui/src/components/DataQuality/TestSuites/TestSuites.component.tsx

```diff
@@ -177,8 +180,10 @@ export const TestSuites = () => {
   };
 
   useEffect(() => {
-    fetchTestSuites();
-  }, [tab]);
+    if (testSuitePermission?.ViewAll || testSuitePermission?.ViewBasic) {
```

Comment:
same

Reply:
as this is testSuite list, and not any particular detail view, i am checking permission from resource.
<img width="772" alt="image" src="https://github.com/open-metadata/OpenMetadata/assets/71748675/cfa6e807-a664-4484-a370-82ab427ec54e">